content/en/platform/corda/4.10/enterprise/notary/hsm-support.md

```diff
@@ -61,7 +61,7 @@ Add the following entries to your workers node.conf files, replacing the placeho
 ```sh
 cryptoServiceName: "AZURE_KEY_VAULT"
 cryptoServiceConf: "<path to the crypto service configuration file>"
-cryptoServiceTimeout: "<the desired timeout, default: 1s>"
+cryptoServiceTimeout: "<the desired timeout, default: 10s>"
```

Comment:
@paulmoloneyr3 - see the comment I gave last week about switching to 10000 milliseconds